arangod/GeneralServer/CommTask.cpp

```diff
@@ -480,15 +480,23 @@ bool CommTask::handleRequestSync(std::shared_ptr<RestHandler> handler) {
   handler->statistics().SET_QUEUE_START(SchedulerFeature::SCHEDULER->queueStatistics()._queued);
 
   RequestLane lane = handler->getRequestLane();
+  RequestPriority prio = PriorityRequestLane(lane);
+
   ContentType respType = handler->request()->contentTypeResponse();
   uint64_t mid = handler->messageId();
 
   // queue the operation in the scheduler, and make it eligible for direct execution
   // only if the current CommTask type allows it (HttpCommTask: yes, CommTask: no)
   // and there is currently only a single client handled by the IoContext
-  auto cb = [self = shared_from_this(), handler = std::move(handler)]() mutable {
+  auto cb = [self = shared_from_this(), handler = std::move(handler), prio]() mutable {
+    if (prio == RequestPriority::LOW) {
+      SchedulerFeature::SCHEDULER->ongoingLowPriorityTasks() += 1;
```

Comment:
How save are we here against crashes during shutdown?
The MetricsFeature manages the gauges memory.
The Scheduler is started before the MetricsFeature, so removed after.
=> The SCHEDULER may still be available, on shutdown, when this `cb` is invoked.
But the MetricsFeture could be shutdown already.

Reply:
Never mind this the Metrics Feature is actually started before the Greating, the Scheduler is started after greating.

On a side note: I would prefer to drop either startsBefore or startsAfter from features, as following one direction as a maintainer is easy to do, following two is tricky...